src/components/SummarySection.js

```diff
@@ -9,9 +9,9 @@ export class SummarySection extends Component {
   }
 
   render() {
-    const { stepValues: stepValues } = this.props
-    const { summary: summary } = stepValues
-    const { result: result } = summary
+    const { stepValues } = this.props
+    const { summary } = stepValues
+    const { result } = summary
```

Comment:
Until now we write `const {stepValues} = this.props` (without spaces after and before the `{`, I think it's important that we'll write the same everywhere.